internal/provider/resource_vault_cluster.go

```diff
@@ -33,9 +33,11 @@ func resourceVaultCluster() *schema.Resource {
 		Description:   "The Vault cluster resource allows you to manage an HCP Vault cluster.",
 		CreateContext: resourceVaultClusterCreate,
 		ReadContext:   resourceVaultClusterRead,
+		UpdateContext: resourceVaultClusterUpdate,
 		DeleteContext: resourceVaultClusterDelete,
 		Timeouts: &schema.ResourceTimeout{
-			Create:  &createVaultClusterTimeout,
+			Create:  &createUpdateVaultClusterTimeout,
+			Update:  &createUpdateTimeoutDuration,
```

Comment:
Should this be `&createUpdateVaultClusterTimeout` instead of using Consul's value?
```suggestion
			Update:  &createUpdateVaultClusterTimeout,
```

Reply:
Ah woops, good spot! Think I'm gonna rename the Consul values to be more specific too